### PR TITLE
[Notifier] Add SlackOptions::threadTs() to send message as reply

### PR DESCRIPTION
**Pull request by WaylandAce**
| Q             | A
| ------------- | ---
| Branch?       | 5.x
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | no
| License       | MIT
| Doc PR        | no

Allow specify argument [thread_ts](https://api.slack.com/methods/chat.postMessage#arg_thread_ts) to send reply message.

### CODE REVIEW

**Review by OskarStark — APPROVED** [review]
Please add a CHANGELOG entry, thanks

**Comment by WaylandAce** [comment]
> Please add a CHANGELOG entry, thanks

5.3 or 5.4?

**Comment by OskarStark** [comment]
5.4 please

**Review by OskarStark — APPROVED** [review]

File: src/Symfony/Component/Notifier/Bridge/Slack/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.4
+---
+
+* Add SlackOptions.threadTs to send message as reply.
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Notifier/Bridge/Slack/CHANGELOG.md]: missing starting space + no final dot please:
```
 * Add `SlackOptions::threadTs()` to send message as reply
```

File: src/Symfony/Component/Notifier/Bridge/Slack/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.4
+---
+
+* Add `SlackOptions::threadTs()` to send message as reply
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Slack/CHANGELOG.md]: Space before * still missing

**Comment by nicolas-grekas** [comment]
Thank you @WaylandAce.

**Pull request merged**